DOAX-VenusVacation/d3dx.ini

```diff
@@ -1369,38 +1401,135 @@ run = CommandListClothes
 ; Bytecode hash: 8479a35b
 hash = 5af6c9fd4d23c6e0
 run = CommandListClothes
+[ShaderOverrideClothesPS8-20200610]
+; Bytecode hash: 7a8213e2
+hash = dd9678efa5efd982
+run = CommandListClothes
 
-[ShaderOverrideClothesPS9]
-; Cherry Blossoms Heart
+[ShaderOverrideClothesPS9-Old]
+; Cherry Blossoms Heart, Kasumi Martini Gemstone, Martini SR Hardware
 ; Bytecode hash: b0adc5a2
 hash = e6a0802f6b5a0285
 run = CommandListClothes
-[ShaderOverrideClothesPS10]
-; Common Mermaid | Martini SR SSR
+[ShaderOverrideClothesPS9-20200610]
+; Bytecode hash: 72899d57
+hash = 52c5ae3cdc7810ca
+run = CommandListClothes
+
+[ShaderOverrideClothesPS10-Old]
+; Common Mermaid | Martini SR SSR | Reindeer Bell
 ; Bytecode hash: 75e3e462
 hash = 7633276941ec4f7a
 run = CommandListClothes
-[ShaderOverrideClothesPS11]
+[ShaderOverrideClothesPS10-20200610]
+; Bytecode hash: 3ea17e58
+hash = c46741f26dabb5fd
+run = CommandListClothes
+
+[ShaderOverrideClothesPS11-Old]
 ; Transparent white cloth on black bikini
 ; Bytecode hash: c42e8e56
 hash = 746c0cfeba78dbd9
 run = CommandListClothes
+[ShaderOverrideClothesPS11-20200610]
+; Bytecode hash: f6c01e51
+hash = 1a57b0790b876d57
+run = CommandListClothes
 
 [ShaderOverrideClothesPS12-Old]
-; Destiny Child bikini text (?)
+; Destiny Child bikini text, Dusk Set Frills
 ; Bytecode hash: ab5e9fae
 hash = 088290cf9c3bf97e
 run = CommandListClothes
 [ShaderOverrideClothesPS12-20190425]
 ; Bytecode hash: 21cf1e72
 hash = 334a305914e33465
 run = CommandListClothes
+[ShaderOverrideClothesPS12-20200610]
+; Bytecode hash: df34aecb
+hash = 5c4ca403dd607603
+run = CommandListClothes
 
-[ShaderOverrideClothesPS13]
-; Hitomi Jeans SR
+[ShaderOverrideClothesPS13-Old]
+; Hitomi Jeans SR (top of jeans)
 ; Bytecode hash: 1ead111f
 hash = 074cff4a129d961c
 run = CommandListClothes
+[ShaderOverrideClothesPS13-20200610]
+; Bytecode hash: f92db6e8
+hash = 635329c855070942
+run = CommandListClothes
+
+[ShaderOverrideAccessoryLensePS-Old]
+; Glasses and Demist
+; Bytecodde hash: 3da2a1ac
+hash = 1e372bfddbafcfc9
+run = CommandListClothes
+allow_duplicate_hash=overrule

```

Comment:
I'd ideally like to avoid adding overrules in the base mod, though avoiding the warnings during the update may justify it in this case. Is this just gacha demist that it's conflicting with (in which case I can just update that and add the overrule there), or are there other mods you are aware of that use this hash as well? Same question for ShaderOverrideTanPS below.

Originally the duplicate hash warning was to alert modders if they were adding a section for a shader they had already defined elsewhere to help prevent bugs early. For this game that's less of an issue though - our concerns are more performance if CommandListClothes is run multiple times per frame, and perhaps rendering issues (e.g. if a transparent object ends up being drawn multiple times). To that end maybe I should be thinking about limiting the CommandLists to limit themselves to only run once per draw call per frame instead.

Reply:
Agreed, I did it in the temporary patch i included just to avoid too many people complaining about seeing warnings since i moved a bunch of the older hashes into the base mod that they had littered in various ini files.